pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/DataObjectHelperController.php

```diff
@@ -1072,103 +1487,33 @@ public function importProcessAction(Request $request)
         unlink($file);
         rename($tmpFile, $file);
 
-        // prepare mapping
-        foreach ($mappingRaw as $map) {
-            if ($map[0] !== '' && $map[1] && !empty($map[2])) {
-                $mapping[$map[2]] = $map[0];
-            } elseif ($map[1] == 'published (system)') {
-                $mapping['published'] = $map[0];
-            } elseif ($map[1] == 'type (system)') {
-                $mapping['type'] = $map[0];
-            }
-        }
+        $rowId = $skipFirstRow ? $job + 1 : $job;
 
-        // create new object
-        $className = 'Pimcore\\Model\\DataObject\\' . ucfirst($request->get('className'));
-        $parent = DataObject::getById($request->get('parentId'));
+        try {
+            $service = \Pimcore::getContainer()->get('pimcore.object.importconfig');
```

Comment:
`$this->get()` or type hint against importconfig service in action parameters (same as `Request $request`).